qface/idl/listener.py

```diff
@@ -152,6 +152,9 @@ def enterOperationSymbol(self, ctx: TParser.OperationSymbolContext):
         self.operation = Operation(name, self.interface)
         self.parse_annotations(ctx, self.operation)
         self.parse_type(ctx, self.operation.type)
+        modifier = ctx.operationModifierSymbol()
```

Comment:
I'm still not convinced that this is a useful addition to the grammar. I really would like to postpone this change until we understand this more. Adding something to the grammar is very easy, but removing something later is almost impossible.